test/setup.js

```diff
@@ -1,41 +1,33 @@
-/*
- * Copyright © 2018 Lisk Foundation
- *
- * See the LICENSE file at the top-level directory of this distribution
- * for licensing information.
- *
- * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
- * no part of this software, including this file, may be copied, modified,
- * propagated, or distributed except according to the terms contained in the
- * LICENSE file.
- *
- * Removal or modification of this copyright notice is prohibited.
- */
 'use strict';
 
 require('colors');
+var chai = require('chai');
+var sinon = require('sinon');
+var sinonChai = require('sinon-chai');
 var supertest = require('supertest');
+process.env.NODE_ENV = 'test';
 
 // Root object
-var test = {};
+var testSuite = {};
 
-test.config = require('./data/config.json');
-test.config.root = process.cwd();
+testSuite.config = require('./data/config.json');
+testSuite.config.root = process.cwd();
 
 // Optional logging
 if (process.env.SILENT === 'true') {
-	test.debug = function () { };
+	testSuite.debug = function () { };
 } else {
-	test.debug = console.log;
+	testSuite.debug = console.log;
 }
 
 // Node configuration
-test.baseUrl = 'http://' + test.config.address + ':' + test.config.httpPort;
-test.api = supertest(test.baseUrl);
+testSuite.baseUrl = 'http://' + testSuite.config.address + ':' + testSuite.config.httpPort;
+testSuite.api = supertest(testSuite.baseUrl);
 
-test._ = require('lodash');
+// Custom lodash mixin
+testSuite._ = require('lodash');
```

Comment:
Is setting `_` as a global in its own right a possibility or too messy? There's a lot of boilerplate left just extracting it into local scope with `var _ = testSuite._`.